android/src/app/organicmaps/settings/SettingsPrefsFragment.java

```diff
@@ -344,24 +346,24 @@ else if (preference.getKey() != null && preference.getKey().equals(getString(R.s
     return super.onPreferenceTreeClick(preference);
   }
 
-  private void initLangInfoLink()
+  private void initTtsLangInfoLink()
   {
-    if (mLangInfoLink != null)
-    {
-      Spannable link = new SpannableString(getString(R.string.prefs_languages_information_off_link));
-      link.setSpan(new ForegroundColorSpan(ContextCompat.getColor(requireContext(),
-                                                                  UiUtils.getStyledResourceId(requireContext(), R.attr.colorAccent))),
-                   0, link.length(), 0);
-      mLangInfoLink.setSummary(link);
-      String TTS_INFO_LINK = requireActivity().getString(R.string.tts_info_link);
-      mLangInfoLink.setOnPreferenceClickListener(preference -> {
-        final Intent intent = new Intent(Intent.ACTION_VIEW);
-        intent.setData(Uri.parse(TTS_INFO_LINK));
-        requireContext().startActivity(intent);
-        return false;
-      });
-      removePreference(getString(R.string.pref_navigation), mLangInfoLink);
-    }
+    Preference ttsLangInfoLink = getPreference(getString(R.string.pref_tts_info_link));
+    final String ttsLinkText = getString(R.string.prefs_languages_information_off_link);
+    Spannable link = new SpannableString(ttsLinkText + "\u2197");
+    // Set link color.
+    link.setSpan(new ForegroundColorSpan(ContextCompat.getColor(requireContext(),
+                                                                UiUtils.getStyledResourceId(requireContext(), R.attr.colorAccent))),
+                 0, ttsLinkText.length(), 0);
+    ttsLangInfoLink.setSummary(link);
+
+    final String ttsInfoUrl = requireActivity().getString(R.string.tts_info_link);
+    ttsLangInfoLink.setOnPreferenceClickListener(preference -> {
```

Comment:
nit: is there a way to mark the lambda variable as unused in Java? Maybe use _ ?

Reply:
Java doesn't support `_` as an argument name. Let's keep it as `preference`.